tests/vm_networking_test.go

```diff
@@ -159,8 +165,44 @@ var _ = Describe("Networking", func() {
 			case "InboundVM":
 				addr = inboundVM.Status.Interfaces[0].IP
 			}
-			cmdCheck = fmt.Sprintf("ping %s -c 1 -w 5\n", addr)
+
+			// Check br1 MTU inside the pod
+			vmPod := tests.GetRunningPodByLabel(outboundVM.Name, v1.DomainLabel, tests.NamespaceTestDefault)
+			output, err := tests.ExecuteCommandOnPod(
+				virtClient,
+				vmPod,
+				vmPod.Spec.Containers[0].Name,
+				[]string{"ip", "address", "show", "br1"},
+			)
+			log.Log.Infof("%v", output)
+			Expect(err).ToNot(HaveOccurred())
+			// the following substring is part of 'ip address show' output
+			expectedMtuString := fmt.Sprintf("mtu %d", expectedMtu)
+			Expect(strings.Contains(output, expectedMtuString)).To(BeTrue())
+
+			// Check eth0 MTU inside the VM
+			addrShow = "ip address show eth0\n"
 			out, err := expecter.ExpectBatch([]expect.Batcher{
+				&expect.BSnd{S: "\n"},
+				&expect.BExp{R: "\\$ "},
+				&expect.BSnd{S: addrShow},
+				&expect.BExp{R: fmt.Sprintf(".*%s.*\n", expectedMtuString)},
+				&expect.BSnd{S: "echo $?\n"},
+				&expect.BExp{R: "0"},
+			}, 180*time.Second)
+			log.Log.Infof("%v", out)
```

Comment:
Here too. As @vladikr said, `By()` statements are the way to go.

Reply:
It doesn't seem to me the way By()s are supposed to be used allows for debug statements like that. By() is used to describe the intent of a test section but not to dump results of its execution. I agree By()s are needed though.